chain-tx-enclave-next/mls/src/group.rs

```diff
@@ -443,6 +446,37 @@ impl GroupAux {
         let commit_content = CommitContent::new(self.tree.cs, &commit, proposals)
             .map_err(|_| ProcessCommitError::CommitError)?;
 
+        // update kp_secret for self updating proposal
+        let self_update_proposal = commit_content
+            .updates
+            .iter()
+            .filter_map(|(sender, update)| {
```

Comment:
can use find? because it's finding the first one?

Reply:
done